src/styles/_global-styles.scss

```diff
@@ -1,3 +1,7 @@
+:root {
+    --green1: #4de0b4;
```

Comment:
Because this is specific to Klaro, we should add a comment here to say something like:
```
// Specify a different green color for for Klaro cookie notice. ONLY used for Klaro.
```

Either that, or we see if there's a way to move this color into the `.klaro` classes below... that may mean not using the variable though and just overriding the CSS  colors directly.

Basically though, we need to document why this `--green1` exists, so that other developers do NOT use it in other areas of DSpace.